inginious/frontend/templates/course_admin/task_list.html

```diff
@@ -113,4 +118,48 @@ <h5 class="modal-title" id="taskDispenserModalLabel">{{_("Switch task dispenser"
         </button>
     </div>
 </div>
+
+<script>
+	const tasksWithErrors = document.getElementsByClassName("bg-danger")
+
+	for(let task of tasksWithErrors){
+		document.getElementById("errorLogContainer").innerHTML += "<br>&nbsp &nbsp &nbsp<a href='#" + task.id + "' style='color: white; text-decoration: underline'>" + task.innerText + "</a><br>"
+	}
+
+	console.log(tasksWithErrors.length)
+
+	if(tasksWithErrors.length == 1){
+		document.getElementById("errorLog").innerText = "There is a task with some errors";
+		document.getElementById("errorLogContainer").classList.remove("d-none");
+		document.getElementById("errorLogContainer").classList.add("bg-danger"); 
+	} else if(tasksWithErrors.length > 0){
+		document.getElementById("errorLog").innerText = "There are " + tasksWithErrors.length + " tasks with some errors";
+		document.getElementById("errorLogContainer").classList.remove("d-none");
+		document.getElementById("errorLogContainer").classList.add("bg-danger");
+	} else {
+		document.getElementById("errorLogContainer").classList.add("d-none");
+	}
+
+	const list_delete_button = document.getElementsByClassName("delete_task")
+	
+	for(let del_button of list_delete_button){
+		const task_id = del_button.parentElement.parentElement.id
+		
+		del_button.addEventListener("click", () => {
```

Comment:
You have to be very careful here. Deleting tasks from the list is the responsibility of each task dispensers, and button class can differ from implementations. I think this piece of code can be added in the ``task_dispenser_delete_task`` function which has to be called by each task dispenser when a task in deleted.